CONTRIBUTING.md

```diff
@@ -0,0 +1,46 @@
+# Contributing to the W3C Process Document
+Thank-you for offering to contribute to the [W3C Process Document](https://www.w3.org/2017/Process-20170301/). Here we provide some details on how you can contribute. 
+
+Changes to the process document tend to fall into a few classes, and how the W3C handles these varies. Simple editorial changes may require little review, but more substantial changes will require a more substantial review and discussions by the W3C community. Please read the below, and get involved as explained. 
+
+If you have any questions, please [raise an issue](/w3c/w3process/issues).
```

Comment:
Using relative path here will cause 404. I'd suggest adding the prefix (https://github.com) here.

(Same below.)